### PR TITLE
Make sure subscriptions have nice JSON output

### PR DESCRIPTION
**Pull request by biilmann**

### CODE REVIEW

**Comment by rybit** [comment]
Some minor issues. And could you put a json ignore tag on deleted at?

File: models/subscriptions.go [hunk 1]
```diff
@@ -11,17 +11,18 @@ import (
 )
 
 type Subscription struct {
-	ID   string `gorm:"unique;primary"`
-	Type string
+	ID        string `gorm:"unique;primary",json:"id"`
+	Type      string `json:"type"`
 
-	User     *User  `json:"user,omitempty"`
+  User     *User  `json:"user,omitempty"`
```
Comment (rybit) [inline models/subscriptions.go]: ??

Comment (biilmann) [reply models/subscriptions.go]: Interesting - was a small attempt at how GitHub's new merge conflict tool works. Apparently it doesn't :P

Comment (biilmann) [reply models/subscriptions.go]: Fixed

**Comment by rybit** [comment]
lgtm

**Pull request merged**